app/services/person_update_notifier.rb

```diff
@@ -21,6 +21,7 @@ def self.send_reminder person, within
     person.reload
     if send_update_reminder? person, within
       ReminderMailer.person_profile_update(person).deliver_later
+      person.skip_must_have_team = true
```

Comment:
@willmcb,  my thought for this line is to change the order:
put the update before sending email,  it sounds not ideal, but the reason for this is 
 -  the email using 'deliver_later',  if anything wrong, the task related to email will be handled outside this cronjob, so it will add a task into queue no matter whether what will happen to it, so it doesn't matter the timestamp is updated before or after
-  not sure if anything happen to person.update,  how the error will be handled, my guess (and my concern) it may end this job with error, then this job will be re-triggered again ( didn't test my though), so ideally we should catch individual error, but if this happen,  at least, the email won't be resent again. 

But, it is just my thought,  no need to block this fix